pkg/datapath/linux/config/config.go

```diff
@@ -224,6 +224,10 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["ENABLE_PREFILTER"] = "1"
 	}
 
+	if option.Config.SkipSocketLBInPodNamespace {
```

Comment:
It'll be good to document this option, and the use cases.

Reply:
maybe we can track it in a separate PR?

Reply:
Yes, specifically in `Documentation/gettingstarted/istio.rst` since that's the target use for this new flag.

Reply:
It'll be nice to add the documentation in the same PR that adds it. But I don't have any strong feelings about this. As long as, the item is tracked and addressed, I'm fine deferring it to a follow-up PR.

Reply:
Added a note in `Documentation/gettingstarted/istio.rst`, PTAL :)